Sources/PackageDescription/PackageDescription.swift

```diff
@@ -88,11 +98,11 @@ public final class Package {
 
     /// The name to use for C modules.
     ///
-    /// If present, the Swift Package Manager searches for a `<name>.pc` file
-    /// to get the required additional flags for a system target.
+    /// If present, the Swift Package Manager searches for a `<name>.pc` file to
+    /// get the required additional flags for a system target.
     public var pkgConfig: String?
 
-    /// An array of providers for a system target.
+    /// An array of providers for the system target.
```

Comment:
I think it's correct to use the indefinite article "a" here, because there isn't a particular system target that is referred to in context.